imports/api/users/publications.js

```diff
@@ -1,8 +1,33 @@
 /* eslint-disable prefer-arrow-callback */
 
 import { Meteor } from 'meteor/meteor';
+import { SimpleSchema } from 'meteor/aldeed:simple-schema';
+import { Memberships } from '/imports/api/memberships/memberships.js';
 
 // Everyone has access to all of his own stuff automatically
 Meteor.publish(null, function self() {
   return Meteor.users.find({ _id: this.userId });
 });
+
+// Subscribes to the public fields of users in the same community
+Meteor.publish('users.inCommunitybyId', function userInCommunitybyId(params) {
+  new SimpleSchema({
+    _id: { type: String, regEx: SimpleSchema.RegEx.Id },
+  }).validate(params);
+
+  const { communityId } = params;
```

Comment:
Ez itt nem lehet jo. A parameter schema egy _id-t enged a paramater objectben, de itt egy communityId-t is megprobalsz kivenni belole. Az biztos nem lesz benne.
Ergo gondolom a communityId: undefined lesz.
Ha communityId-t is akarsz parameterkent beadni, akkor a schema-ban szerepelni kell, es ugy kell kiolvasni a kettot egyutt, hogy egyszerre:
const { _id, communityId } = params;

Reply:
Javítom